unlock-app/src/components/content/DashboardContent.jsx

```diff
@@ -1,139 +1,16 @@
-import PropTypes from 'prop-types'
-import { AuthenticationContext } from '../../contexts/AuthenticationContext'
-import UnlockPropTypes from '../../propTypes'
-import Layout from '../interface/Layout'
-import Account from '../interface/Account'
-import CreatorLocks from '../creator/CreatorLocks'
-import BrowserOnly from '../helpers/BrowserOnly'
-import { pageTitle } from '../../constants'
-import LoginPrompt from '../interface/LoginPrompt'
-
-import {
-  CreateLockButton,
-  CancelCreateLockButton,
-} from '../interface/buttons/ActionButton'
-import { Phone } from '../../theme/media'
-import { Button } from '@unlock-protocol/ui'
-
-const ButtonToCreateLock = ({ formIsVisible, toggleForm }) => {
-  const { account } = useContext(AuthenticationContext)
-
-  return (
-    <>
-      {formIsVisible && (
-        <Button
-          variant="outlined-primary"
-          id="CreateLockButton"
-          onClick={toggleForm}
-        >
-          Cancel Lock
-        </Button>
-      )}
-      {!formIsVisible && (
-        <Button disabled={!account} id="CreateLockButton" onClick={toggleForm}>
-          Create Lock
-        </Button>
-      )}
-    </>
-  )
-}
-
-ButtonToCreateLock.propTypes = {
-  formIsVisible: PropTypes.bool,
-  toggleForm: PropTypes.func.isRequired,
-}
-
-ButtonToCreateLock.defaultProps = {
-  formIsVisible: false,
-}
+import { useRouter } from 'next/router'
 
 export const DashboardContent = () => {
-  const { account, network } = useContext(AuthenticationContext)
-
-  const [formIsVisible, setFormIsVisible] = useState(false)
-  const toggleForm = () => {
-    formIsVisible ? setFormIsVisible(false) : setFormIsVisible(true)
-  }
-  const hideForm = () => {
-    setFormIsVisible(false)
-  }
+  const router = useRouter()
 
-  return (
-    <Layout title="Creator Dashboard">
-      <Head>
-        <title>{pageTitle('Dashboard')}</title>
-      </Head>
-      {!account && (
-        <LoginPrompt>
-          In order to deploy locks, we require the use of your own Ethereum
-          wallet.{' '}
-          <a
-            target="_blank"
-            rel="noreferrer"
-            href="https://ethereum.org/en/wallets/"
-          >
-            Learn more about wallets
-          </a>
-        </LoginPrompt>
-      )}
-      {account && (
-        <BrowserOnly>
-          <div className="flex flex-col gap-3 md:items-center md:flex-row">
-            <Account />
-            <div className="md:ml-auto">
-              <ButtonToCreateLock
-                toggleForm={toggleForm}
-                formIsVisible={formIsVisible}
-              />
-            </div>
-          </div>
-          <Phone>
-            <Warning>
-              The Dashboard is currently not optimized for a mobile experience.
-              To create locks, please use a desktop computer.
-            </Warning>
-          </Phone>
+  useEffect(() => {
+    // force redirect to the new dashboard
+    router.push('/locks')
+  }, [router])
```

Comment:
Is there any need for re-direct? let's remove it and make locks the index path as well?

Reply:
for any eventual external link

Reply:
I think you misunderstood me here. 

I was saying we should simply make /locks available at /dashboard and remove the dashboard content.

Reply:
so keep `/dashboard` instead of `/locks`? 

i think the `/locks` is better here

Reply:
You can keep both but since the locks page was dashboard, it makes sense to put it there instead of a redirect which adds friction to user experience. 

CC @julien51

Reply:
I like the idea of using a more explicit `/locks` vs `/dashboard`... and I think it's better to have a single path.
That said I want to understand more about the friction? I would assume redirects are pretty painless, but maybe I am missing something?

Reply:
redirects add delay to user experience. If we can do without them, /locks or /dashboard either would be fine.

Since `/dashboard` was in use before and have linked elsewhere, I proposed replacing it.

Reply:
removed the redirect and replaced the content inside `DashboardContent` page